doc/source/development/contributing_codebase.rst

```diff
@@ -613,22 +613,15 @@ Testing involving network connectivity
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 It is highly discouraged to add a test that connects to the internet due to flakiness of network connections and
-lack of ownership of the server that is being connected to. If network connectivity is absolutely required, use the
-``tm.network`` decorator.
+lack of ownership of the server that is being connected to. If network connectivity is absolutely required, mock
```

Comment:
With this, are these still highly discouraged? Seems like all the reasons to discourage them no longer apply.

Reply:
Good point I'll clarify the wording here